conditional/blueprints/attendance.py

```diff
@@ -437,23 +436,23 @@ def get_seminar_attendees(meeting_id):
         total_pages = (int(len(all_cm) / 10) + 1)
     else:
         total_pages = (int(len(all_cm) / 10))
-    return render_template(request,
-                       'attendance_history.html',
-                       username=user_name,
-                       history=all_meetings,
-                       pending_cm=pend_cm,
-                       pending_ts=pend_ts,
-                       num_pages=total_pages,
-                       current_page=int(page))
+    return render_template('attendance_history.html',
+                           username=username,
```

Comment:
This should be split across two commits. One change to remove the `request` parameter, the other to change the indentation.